extensions/file/src/main/java/org/mule/extension/file/internal/command/LocalWriteCommand.java

```diff
@@ -64,7 +65,7 @@ public void write(String filePath, InputStream content, FileWriteMode mode,
       throw new FileAccessDeniedException(format("Could not write to file '%s' because access was denied by the operating system",
                                                  path),
                                           e);
-    } catch (org.mule.extension.file.common.api.exceptions.FileAlreadyExistsException e) {
```

Comment:
why this? Is there a test which verifies that the associated error type is still catchable?